uefi-pi/src/pi/fv.rs

```diff
@@ -266,3 +290,25 @@ impl CommonSectionHeader2 {
         unsafe { &*slice_from_raw_parts(self as *const Self as *const u8, size_of::<Self>()) }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn test_fvh_checksum() {
```

Comment:
Could you add tests for:

- A `FirmwareVolumeHeader` with an existing checksum to ensure the checksum is ignored.
- Calling `update_checksum()` multiple times to ensure it has no effect on the calculated checksum.